benchmarks/src/main/java/io/improbable/keanu/benchmarks/ElementwiseBinaryDoubleOperations.java

```diff
@@ -0,0 +1,44 @@
+package io.improbable.keanu.benchmarks;
+
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+@State(Scope.Benchmark)
+public class ElementwiseBinaryDoubleOperations {
+
+    private static final int NUM_OPERATIONS = 100;
```

Comment:
Might as well make this a `1000` like you do in `BroadcastBinaryDoubleOperations`. And add a few more key operations seeing as it's as easy as adding them to the operation param. Divide and subtract?

Reply:
Just removed the loop - it's not great for us to do loops inside of benchmarks anyway.  I'll delete in both locations.

Re: extra ops - I think that's just burning CPU for the sake of it tbh - that would just be measuring the CPU performance of performing those floating point ops rather than measuring anything about our code IMO.